src/store/reducers/stateCache/stateCache.types.ts

```diff
@@ -32,6 +34,7 @@ interface IncomingRequestProps {
   logo?: string;
   label?: string;
   event?: KeriaNotification;
+  peerConnectionEvent?: PeerConnectSigningEvent;
   multisigIcpDetails?: MultiSigIcpRequestDetails;
   signTransaction?: SignTransaction;
```

Comment:
Also here, we have `signTransaction` - those types should be removed and embed enough information within `PeerConnectSigningEvent`.